### ISSUE TITLE
Add email field in the user creation form

### ISSUE THREAD

**Issue by RadheTians** [openMF/digital-bank-ui#77]
#### Feature Request

I have not seen an email id or contact number field in the user creation form and it's essential to implement a reset-password link functionality on user email or send SMS on user contact number.

#### What is the motivation/use case for changing the behavior?

It's required while sending notifications and reset password link to the user email.

#### New Idea will look like the below screenshot.
 

![Screenshot from 2020-12-15 01-21-18](https://user-images.githubusercontent.com/50411704/102128606-20a8e480-3e74-11eb-8515-9c31c59bc693.png)

**Comment by RadheTians** [comment]
@abhi40308 @giorgiozoppi what you guys think, Should I go ahead with it?

**Comment by abhi40308** [comment]
Email notifications are not implemented in backend. Without it being in backed, we shouldn't include in the client for now.

**Comment by giorgiozoppi** [comment]
@RadheTians  we might need some integration testing with taiko before implementing new features, we want to stabilize and improve the UX of what we have. For looking at taiko https://github.com/getgauge/taiko. Can i create the ticket and assign to you?

**Comment by RadheTians** [comment]
Sure @giorgiozoppi I'm not familiar with taiko. So, I might take some time to get familiar myself with taiko.

**Comment by RadheTians** [comment]
@abhi40308 if I'm not wrong then it's required to implement atleast reset password-link featue in login form.

**Comment by abhi40308** [comment]
@RadheTians  Yes we need reset password feature, but it requires some backend features which are not present at the current moment, as @giorgiozoppi mentioned.

**Comment by RadheTians** [comment]
Okay, we leave it until then.

**Comment by mittulmandhan** [comment]
@giorgiozoppi @abhi40308 I would like to implement integration testing with taiko

**Comment by Darkcoder011** [comment]
can I take This Issue I Work IN More Than Many Web Project
